ABOS/lftp_sync.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+
+source $DATA_SERVICES_DIR/env
+
+declare -r FTP_SOURCE=$IMOS_PO_CREDS_BOM_FTP_ADDRESS
+declare -r FTP_USER=$IMOS_PO_CREDS_BOM_FTP_USERNAME
+declare -r FTP_PASSWORD=$IMOS_PO_CREDS_BOM_FTP_PASSWORD
+declare -r YEAR=`date +%Y`
+declare -r FTP_DIR="/register/bom404/outgoing/IMOS/MOORINGS/$YEAR"
+FTP_EXTRA_OPTS=""
+
+declare -r ABOS_SOFS_WIP_DIR=$WIP_DIR/ABOS/SOFS
+declare -r ABOS_SOFS_WIP_DIR_FTP=$ABOS_SOFS_WIP_DIR/ftp
+declare -r ABOS_SOFS_INCOMING_DIR=$INCOMING_DIR/ABOS/SOFS
+
+# sync files from remote AATAMS ftp server
+# $1 - lftp log file
+sync_files() {
+    local lftp_log_file=$1; shift
+    lftp -e "open -u $FTP_USER,$FTP_PASSWORD $FTP_SOURCE; mirror -e --parallel=10 $FTP_EXTRA_OPTS --exclude-glob *_ODV.zip -vvv --log=$lftp_log_file $FTP_DIR $ABOS_SOFS_WIP_DIR_FTP; quit"
```

Comment:
I don't think we need `--exclude-glob *_ODV.zip`. Instead, we should include only netCDF files and directories, i.e. `--include-glob='*.nc' --include-glob='*/'`